### DISCUSSION TITLE
Using LogLevel to debug

### DISCUSSION THREAD

**Discussion by lexahall** [petyosi/react-virtuoso#740] [Q&A] [answered]
In the [Troubleshooting](https://virtuoso.dev/troubleshooting) section, there is a reccomendation to use set "the logLevel property to LogLevel.DEBUG or by set a globalThis.VIRTUOSO_LOG_LEVEL to LogLevel.DEBUG", I'm a little unsure where to set this. I don't see a logLevel prop directly on the <Virtuoso> component, and am unsure where I would set the global var. Can anyone give me a little more context here? 

Thanks!

**Comment by masbagal** [comment, marked as answer]
Set it on the Virtuoso component. 
```js
import { Virtuoso, LogLevel } from "react-virtuoso";

 <Virtuoso
     logLevel={LogLevel.DEBUG}
     ...
/>
```

I also don't see the `logLevel` props, but after adding that, it works and showing logs on my browser console.

The log will look like this:

<img width="387" alt="Screen Shot 2022-09-14 at 11 56 39" src="https://user-images.githubusercontent.com/16324649/190063054-6372e094-42db-4240-b566-4b3b07a07714.png">
